cwe_api/inferencer.py

```
@@ -0,0 +1,50 @@
+    output = pipe(vuln, batch_size=2, truncation="only_first")
+    print("output: ",output)
+
+    softmax_output = m(output[0])[0]
+    ind = np.argpartition(softmax_output, -NUMBER_OF_PREDICTIONS)[-NUMBER_OF_PREDICTIONS:]
+
+    reversed = np.flip(ind.numpy(),0).copy()
+    score = softmax_output[reversed]
+
+    predicts = [{'priority': i, 'label': id2label[str(reversed[i])], 'score': float(score[i].numpy())} for i in range(0, 3)]
+
+    return predicts
```

Ruff

Unnecessary assignment to `predicts` before `return` statement

Remove unnecessary assignment

(RET504)